handlers_test.go

```diff
@@ -26,6 +26,12 @@ func TestIndexHandler(t *testing.T) {
 
 	// test the body of the index
 	testBodyContains(t, `<h1>Welcome to yCTF</h1>`, w.Body.String())
+
+	// test the session cookie named yctf-session is set
+	testCookieContains(t, "yctf-session", w.Header()["Set-Cookie"])
```

Comment:
For consistency with `handlers.flag2_test.go`, should this instead use `w.Results().Cookies()`? We would have the `*http.Cookie` objects instead. The function signature for `testCookieContains` might have to change - and `handlers.flag2_test.go` might be able to make use of `testCookieContains()` as well.